doc/optparse/tutorial.rdoc

```diff
@@ -367,6 +373,106 @@ Executions:
 
 Omitting an optional argument does not raise an error.
 
+=== Argument Values
+
+Permissible argument values may be restricted
+either by specifying explicit values
+or by providing a pattern that the given value must match.
+
+==== Explicit Argument Values
+
+You can specify argument values in either of two ways:
+
+- Specify values an array of strings.
+- Specify values a hash.
+
+===== Explicit Values in Array
+
+You can specify explicit argument values in an array of strings.
+The argument value must be one of those strings, or an unambiguous abbreviation.
```

Comment:
I think I just learned something. I was missing a word like explicitly permitted or the like.

Reply:
This was how I came to misread it: I'd missed that the setting was an "allowed values" list.